Safety/AttitudeManager/Inc/attitudeStateClasses.hpp

```diff
@@ -4,12 +4,12 @@
 #include "attitudeManager.hpp"
 #include "AttitudeDatatypes.hpp"
 #include "attitudeConfig.hpp"
+#include "../../Common/Inc/AttitudePathInterface.hpp"
+#include "../../Common/Inc/CommWithPathManager.hpp"
```

Comment:
Why did you need to step back two directories here... shouldn't the `Common/Inc/*` folder be found by Safety's build system?

Reply:
You're right, that's fixed